src/EduManagementLab.Api/Controllers/CoursesController.cs

```diff
@@ -149,6 +149,21 @@ public ActionResult<CourseDto> AddCourseMembership(Guid courseId, Guid userId, D
 
         }
 
+        [HttpPut]
+        [Route("{courseId}/UpdateMembership/{userId}")]
+        public ActionResult<Course> UpdateCourseMembershipEnrolledDate(Guid courseId, Guid userId, UpdateMembershipEnrolledDateModel MembershipEnrolledDate)
```

Comment:
Compile error.
UpdateMembershipEnrolledDateModel is missing.
Should it be a DateTime?

Reply:
yes. It's DatTime.